index.html

```diff
@@ -14,17 +14,21 @@
       src="https://kit.fontawesome.com/04970764d6.js"
       crossorigin="anonymous"
     ></script>
-    <link rel="stylesheet" media="(min-width: 992px)" href="media.css">
-    <link rel="stylesheet" media="(max-width: 992px)" href="style.css">
+    <link rel="stylesheet" media="(min-width: 992px)" href="/desktop.css">
+    <link rel="stylesheet" media="(max-width: 992px)" href="/mobile.css">
```

Comment:
Hello @NatiG25 + @Mehdi-Rh, you have done a great job on this project! However, I have found a problem with the css styles of the project not loading as you can see below 👇🏻.

https://user-images.githubusercontent.com/4247740/151851418-0530217d-d1ad-4492-b5e0-af69c7cea6d5.mp4

- [ ] I  also suggest that you remove the slashes before the css files `<link href="/desktop.css">` to `<link href="desktop.css">` and `<link href="/mobile.css">` to `<link href="mobile.css">`

### Also try to check if your project meets the following project requirements 😄 

- [ ] On desktop size the hamburger menu button is not visible.
- [ ] On mobile the hamburger menu button is visible.
- [ ] When the user clicks (or taps) the hamburger button, the mobile menu appears.
- [ ] When the user clicks (or taps) the close (X) button, the mobile menu disappear.
- [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- [ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu disappear.
- [ ] The hamburger button and menu follows the design from the chosen template in Figma (e.g., font, colors, images, tests, margins).